GVFS/GVFS.Hooks/Program.cs

```diff
@@ -93,6 +95,26 @@ private static void RunPreCommands(string[] args)
             }
         }
 
+        private static void RunPostCommands(string[] args)
+        {
+            RemindUpgradeAvailable();
+        }
+
+        private static void RemindUpgradeAvailable()
+        {
+            // The idea is to generate a random number between 0 and 100. To make 
+            // sure that the reminder is displayed only 10% of the times a git 
+            // command is run, check that the random number is between 0 and 10, 
+            // which will have a probability of 10/100 == 10%.
+            int reminderFrequency = 10;
+            int randomValue = random.Next(0, 100);
+
+            if (randomValue <= reminderFrequency && ProductUpgrader.IsLocalUpgradeAvailable(GVFSHooksPlatform.GetInstallerExtension()))
+            {
+                Console.WriteLine(Environment.NewLine + GVFSConstants.UpgradeVerbMessages.ReminderNotification);

```

Comment:
I haven't tested rigorously, but since this went down to a random 10%, I don't think I've ever actually seen this message. The code looks good but it would be great to confirm this is all working.

Reply:
Tested this.

<img width="557" alt="screen shot 2018-10-15 at 2 05 02 pm" src="https://user-images.githubusercontent.com/378580/46969322-af96f800-d083-11e8-99cb-6e98dd1d3f9f.png">